Duplicati/Library/Backend/Jottacloud/Jottacloud.cs

```diff
@@ -146,6 +162,22 @@ public Jottacloud(string url, Dictionary<string, string> options)
             m_url_device = JFS_ROOT + "/" + m_userInfo.UserName + "/" + m_device;
             m_url        = m_url_device + "/" + m_mountPoint + "/" + m_path;
             m_url_upload = JFS_ROOT_UPLOAD + "/" + m_userInfo.UserName + "/" + m_device + "/" + m_mountPoint + "/" + m_path; // Different hostname, else identical to m_url.
+
+            m_threads = options.ContainsKey(JFS_THREADS) ? int.Parse(options[JFS_THREADS]) : 4;
+
+            if (!options.TryGetValue(JFS_CHUNKSIZE, out var tmp))
+            {
+                tmp = JFS_DEFAULT_CHUNKSIZE;
+            }
+
+            var chunksize = Utility.Sizeparser.ParseSize(tmp, "mb");
+
+            if (chunksize > int.MaxValue || chunksize < 1024)
```

Comment:
I think `chunksize` is a `long` here.  What's the reasoning for comparing it to `int.MaxValue`?

Reply:
System.IO.BinaryReader.ReadBytes(int count) only seems to accept ints, hence the limitation of max chunk size.